src/test/java/com/bearcat/eventplanner/EventListingScreenTest.java

```diff
@@ -0,0 +1,33 @@
+package com.bearcat.eventplanner;
+
+import com.bearcat.bearcatwebpage.src.EventListingScreen;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.swing.*;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventListingScreenTest {
+
+    private EventListingScreen eventListingScreen;
+
+    @BeforeEach
+    void setUp() {
+        eventListingScreen = new EventListingScreen();
+    }
+
+    @Test
+    void testUpdateEventTextArea() {
+        JTextArea eventTextArea = eventListingScreen.getEventTextArea();
+        List<String> events = eventListingScreen.getEvents();
+
+        StringBuilder expectedText = new StringBuilder();
+        for (String event : events) {
+            expectedText.append(event).append("\n");
+        }
+
+        assertEquals(expectedText.toString(), eventTextArea.getText());
+    }
```

Comment:
This covers the world of the 7 Deadly Sins and Technical Debt: "having no or low code coverage by unit tests, especially in complex part of the program" https://www.methodsandtools.com/tools/tools.php?sonar#:~:text=In%20the%20Sonar%20team%2C%20we%20like%20to%20call,bugs%207%20having%20a%20spaghetti%20design%20%28package%20cycles...%29. 

I added this Unit test to add coverage to the program and ensure proper coverage of the application JFrame as it runs in the testing environment.